src/google/protobuf/compiler/objectivec/objectivec_file.cc

```diff
@@ -239,16 +247,26 @@ void FileGenerator::GenerateHeader(io::Printer* printer) {
       "\n",
       "google_protobuf_objc_version", StrCat(GOOGLE_PROTOBUF_OBJC_VERSION));
 
-  // #import any headers for "public imports" in the proto file.
+  // The bundled protos (WKTs) force off the use of forward declarations.
```

Comment:
remove "off"

Reply:
rephrased, but it is off, they force imports instead of fwd decls.